R/helper_clean_model_data.R

```diff
@@ -31,7 +31,22 @@ clean_model_data <- function(data, datargs, estimator = "") {
 
   for (da in to_process) {
     name <- sprintf(".__%s%%%d__", da, sample.int(.Machine$integer.max, 1))
-    m_formula_env[[name]] <- eval_tidy(mfargs[[da]], data = data)
+    if (da == "fixed_effects") {
```

Comment:
Move this test / conversion below the loop, near the formula conversion. Also consider vapply instead of sapply.

Reply:
This returns a matrix if there are multiple fixed effects.

Reply:
use args_ignored as above.

Reply:
I have to rewrite this a bit, because we actually don't want to ignore "subset" (and we don't actually ignore "args_ignored" right now).